includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -3136,6 +3139,35 @@ class AMP_Allowed_Tags_Generated {
 					),
 				),
 			),
+			array(
+				'attr_spec_list' => array(
+					'data-href' => array(
+						'mandatory' => true,
+					),
+					'media' => array(),
+					'noloading' => array(
+						'value' => array(
+							'',
+						),
+					),
+				),
+				'tag_spec' => array(
+					'amp_layout' => array(
+						'supported_layouts' => array(
+							6,
+							2,
+							3,
+							7,
+							1,
+							4,
+						),
+					),
+					'requires_extension' => array(
+						'amp-facebook',
+					),
+					'spec_name' => 'amp-facebook-comments 1.0',
+				),
+			),
```

Comment:
See https://github.com/ampproject/amphtml/pull/35395#issuecomment-897104502

Reply:
We need to add a new `bento` boolean flag to each `tag_spec` that `requires_extension` so we can easily differentiate between Bento and non-Bento specs.

Reply:
Or rather, `requires_extension` should be a mapping from component to the version(s). In this way, we could supply to components and versions that we are wanting and it can automatically omit tag specs that don't have a matching version.